core/modules.rs

```diff
@@ -557,9 +831,28 @@ impl ModuleMap {
       let import_specifier = module_request
         .get_specifier()
         .to_rust_string_lossy(tc_scope);
+
+      let import_assertions = module_request.get_import_assertions();
+
+      // For static imports, assertions are triples of (keyword, value and source offset)
+      let assertions = parse_import_assertions(tc_scope, import_assertions, 3);
+
+      // FIXME(bartomieju): there are no stack frames if exception
```

Comment:
Is this something to be fixed in this review or later?

Reply:
Later, it's not clear how to fix it